routesrv/options.go

```diff
@@ -0,0 +1,111 @@
+package routesrv
+
+import (
+	"regexp"
+	"time"
+
+	"github.com/zalando/skipper/dataclients/kubernetes"
+	"github.com/zalando/skipper/eskip"
+)
+
+// Options for initializing/running RouteServer
+type Options struct {
+	// Network address that routesrv should listen on.
```

Comment:
This and all below was copied straight from the main `skipper.Options` as-is to avoid diverging the common options too much. Would we want to address godoc for both places in scope of this PR?